utils/zerovec/src/zerovec/mod.rs

```diff
@@ -247,6 +250,19 @@ where
         Ok(Self::Borrowed(slice))
     }
 
+    /// Uses a `&[u8]` buffer as a `ZeroVec<T>` without any verification.
+    ///
+    /// # Safety
+    ///
+    /// `bytes` need to be an output from [`ZeroSlice::as_bytes()`], and `size_of`
+    /// is `mem::size_of::<T::ULE>()`.
```

Comment:
issue: are we not allowed to do that in const yet? bummer.

Reply:
`size_of` appears to be const?